openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+                                "resourceEnvIPConditionValue": "リソース / 環境 / IP アドレス"
+                            }
+                        },
+                        "Session": {
+                            "title": "アクティブセッション時間",
+                            "props": {
+                                "maxSessionTime": "最大セッション時間",
+                                "terminateSession": "セッションの終了"
+                            }
+                        },
+                        "SessionProperty": {
+                            "title": "現在のセッションプロパティ",
+                            "props": {
+                                "ignoreValueCase": "大文字小文字を区別しない",
+                                "properties": "プロパティ"
+                            }
+                        },
+                        "SimpleTime": {
+                            "title": "時間 (曜日、日付、時刻、タイムゾーン)",
+                            "props": {
+                                "startTime": "開始時刻",
+                                "endTime": "終了時刻",
+                                "startDay": "開始曜日",
+                                "endDay": "終了曜日",
+                                "startDate": "開始日付",
+                                "endDate": "終了日付",
+                                "enforcementTimeZone": "タイムゾーン",
+                                "enterTimeZone": "タイムゾーンの入力"
+                            },
+                            "helperTitle": "時間条件",
+                            "helperContent": "次の1つ以上の値を含める必要があります。: 時刻(開始と終了)、曜日(開始と終了)、日付(開始と終了)、および/またはタイムゾーン。"
+                        },
+                        "Script": {
+                            "title": "スクリプト",
+                            "props": {
+                                "scriptId": "スクリプト名"
+                            },
+                            "placeholder": "スクリプトから選択"
+                        },
+                        "ipHelperTitle": "IP 条件",
+                        "ipHelperContent": "開始 IP、終了 IP またはDNS名の少なくとも一つを入力してください。単一の IP アドレスと一致させるには、開始 IP または終了 IP のどちらかに入力します。開始 IP は、終了 IP 以下の値でなければなりません。DNS名は、明示的な一致かまたは最も低いサブドメインのワイルドカードを使用することができます。",
+                        "legacyFormat": "この条件は従来の形式であり、置換する必要があります。",
+                        "nothingSelected": "何も選択されていません",
+                        "attributes": {
+                            "object": "キー:値[,値2,...]"
+                        }
+                    }
+                },
+                "list": {
+                    "grid": ["名前", "状態"],
+                    "addNew": "新しいポリシーの追加",
+                    "description": "このポリシーセットに適用される認可ポリシーを作成、編集、削除します。",
+                    "noPolicies": "このポリシーセットにポリシーを追加していません。"
+                }
+            },
+
+            "resourceTypes": {
+                "list": {
+                    "add": "新規リソースタイプ",
+                    "title": "リソースタイプ",
+                    "description": "適用されるポリシーリソースとそれらのリソースで実行できるアクション用のテンプレートを定義します。",
```

`ポリシーが適用されるリソースと～`

---

I think that it is a reasonable proposal, thank you.
```suggestion
                    "description": "ポリシーが適用されるリソースと～とそれらのリソースで実行できるアクション用のテンプレートを定義します。",
```